### ISSUE TITLE
Worker Compatibilty

### ISSUE THREAD

**Issue by radarippa** [jwinterm/monerospelunker#3]
Does this support workers? App changes config file automatically.

**Comment by jwinterm** [comment]
No, I don't think it supports workers. It should save settings to config
file when you close.
